test/Removal.t.sol

```diff
@@ -429,6 +429,54 @@ contract Removal_multicall is UpgradeableMarket {
   }
 }
 
+contract Removal_market_balance is UpgradeableMarket {
```

Comment:
```suggestion
contract Removal_getMarketBalance is UpgradeableMarket {
```

Reply:
naming convention thus far is 
`ContractName_MethodName(_variant)`

test names are 
`test // base case`
`test(_variant)`